blocks/rich-text/index.js

```diff
@@ -558,6 +560,21 @@ export class RichText extends Component {
 		if ( keyCode === BACKSPACE ) {
 			this.onChange();
 		}
+
+		this.scrollToCaret();
+	}
+
+	scrollToCaret() {
+		const caretRect = this.getEditorSelectionRect();
+		const caretHeight = caretRect.y;
```

Comment:
Is it accurate to label this as a height value? Based on `.y`, I'd think it's a point offset.

Reply:
Actually, what isn't accurate is to grab `.y`, since

> [1] IE and Edge support the non-standard MSDN: ClientRect which does not define x and y properties ([#](https://developer.mozilla.org/en-US/docs/Web/API/DOMRect))

in non-IE, `.y` and `.height` hold the same value for `caretRect`, so I'll just switch the property. Thanks for asking.

Reply:
Edit: I'd written `height` when I meant `top` 🙈 . Updated.